package.json

```diff
@@ -73,13 +72,13 @@
         "release:latest": "DIST=latest BRANCH=master ./scripts/release.sh",
         "release:next": "DIST=next BRANCH=next ./scripts/release.sh",
         "release:cdn": "yarn setup; node ./scripts/prod.js",
-        "setup": "yarn install --frozen-lockfile; npm-run-all clean build:i18n",
+        "setup": "yarn install --frozen-lockfile --ignore-optional; npm-run-all clean build:i18n",
         "start": "npm-run-all setup start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config scripts/webpack.config.js --mode development",
-        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
+        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --openssl-legacy-provider --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
         "start:examples:legacy": "BROWSERSLIST_ENV=production yarn start:examples",
         "start:npm": "yarn setup; yarn build:dev:es",
-        "start:storybook": "LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development start-storybook -p 6061",
+        "start:storybook": "LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6061",
```

Comment:
Same SSL issue as styleguidist. 

Storybook 5.3.6 does not seem to require a specific version of node: https://github.com/storybookjs/storybook/blob/94aca9f154a0ff6862bd8407d6769cbb6c90b1d9/addons/a11y/package.json